### ISSUE TITLE
add UPM manifest for either VPM, or UPM git packages or for submission to OpenUPM

### ISSUE THREAD

**Issue by TayouVR** [Dreadrith/BlendTreeBuilder#1]
I like to have my editor tools in UPM packages whenever possible, so it would be nice if this could be used as one.
Would also make it compatible with the VPM if VRChat wants it.

OpenUPM is a thing I've only recently found out about and started migrating various "legacy" unity packages to. 
Please have a look at it and see if providing this via OpenUPM would be an option.
https://openupm.com/

PS: your other tools as UPM packages would be cool too, I just happened to end up here from your recent tweet

**Comment by Dreadrith** [comment]
Yep yep. VPM compatibility is up there on my to-do list. Expect it soon 👍

**Comment by TayouVR** [comment]
cool!
